hybrasyl/Scripting/Script.cs

```diff
@@ -380,7 +380,7 @@ public bool ExecuteFunction(string functionName, dynamic invoker, dynamic source
                     Compiled.Globals.Set("source", GetUserDataValue(source));
                     if (scriptItem != null)
                         Compiled.Globals.Set("item", GetUserDataValue(scriptItem));
-                    Compiled.Call(Compiled.Globals[functionName]);
+                   return Compiled.Call(Compiled.Globals[functionName]).Boolean;
```

Comment:
`bool` return here is for success/failure of script, not to return internal logic, returning `false` here will have a lot of unwanted effects. also lua can return `nil` and that'll cause an error here